### PR TITLE
Adds new debug verb for highlighting roundstart ATs

### PR DESCRIPTION
**Pull request by coiax**
Debug -> Debug Verbs enable
then
Mapping -> Show roundstart AT markers

Will highlight AT turfs, for easier location and comprehension rather
than looking at the list of coords. List of coords still works fine
though.

### CODE REVIEW

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -154,14 +154,35 @@ GLOBAL_LIST_INIT(admin_verbs_debug_mapping, list(
 	var/dat = {"<b>Coordinate list of Active Turfs at Roundstart</b>
 	 <br>Real-time Active Turfs list you can see in Air Subsystem at active_turfs var<br>"}
 
-	for(var/i=1; i<=GLOB.active_turfs_startlist.len; i++)
-		dat += GLOB.active_turfs_startlist[i]
+	for(var/t in GLOB.active_turfs_startlist)
+		var/turf/T = t
+		dat += "[T.x], [T.y], [T.z]\n"
```
Comment (Cyberboss) [inline code/modules/admin/verbs/mapping.dm]: Use ADMINJUMPCOORD or whatever the macro is

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/mapping.dm (hunk 1 above)
Comment (coiax) [inline code/modules/admin/verbs/mapping.dm]: Does that work if we embed it in the HTML window?

Comment (Cyberboss) [inline code/modules/admin/verbs/mapping.dm]: yeah, it's just an href

Comment (lzimann) [inline code/modules/admin/verbs/mapping.dm]: Yup, works fine in popups.

**Comment by lzimann** [comment]
coiax pls

**Pull request merged**